src/Sulu/Bundle/AdminBundle/Resources/js/components/Checkbox/Checkbox.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+import React from 'react';
+import classnames from 'classnames';
+import Icon from '../Icon';
+import checkboxStyles from './checkbox.scss';
+
+export default class Checkbox extends React.PureComponent {
+    props: {
+        checked: boolean,
+        skin: 'dark' | 'light',
+        className: string,
+        onChange?: (checked: boolean) => void,
+    };
+
+    input: HTMLInputElement;
+
+    static defaultProps = {
+        skin: 'dark',
+        className: '',
+    };
+
+    handleChange = (event: Event) => {
+        if (this.props.onChange && typeof event.target.checked === 'boolean') {
+            this.props.onChange(event.target.checked);
+        }
+    };
+
+    handleButtonClick = () => this.input.click();
+    handleInputClick = (event: Event) => event.stopPropagation();
+    setInput = (input: HTMLInputElement) => this.input = input;
+
+    render() {
+        const classNames = classnames({
+            [checkboxStyles.checkbox]: true,
+            [checkboxStyles.light]: this.props.skin === 'light',
```

Comment:
Why don't we use something like `[checkboxStyles[this.props.skin]]: true`? Then it is easier to add new skins if necessary, and they are validated via flow anyway.